devel/Containerfile.hack

```diff
@@ -0,0 +1,15 @@
+# This Containerfile is targeted to a specialized path where you are changing
+# pure Go code in this project and you're building on the host system with
+# a compatible userspace (e.g. c9s or fedora).  We just take the already
+# built binary and inject it on top of the main upstream container image.
+#
+# Crucially, using this flow imposes no constraints on your build setup,
+# so that adding e.g. `replace github.com/osbuild/images => ../images`
+# will Just Work.
+#
+# To use this, do e.g.:
+#
+# make && podman build --no-cache -t localhost/bib -v $(pwd)/bin:/srcbin -f devel/Containerfile.hack .
```

Comment:
Might need a :z on the volume mount.

Reply:
I don't personally because I now always use podman machine even on Linux, which basically obviates that problem

Reply:
Ok although when we get to virtiofsd it will happen and I think this will not work without machine nor on a Mac and perhaps windows machine, if labeling is supported over the remote connection.

Reply:
Why not just `COPY` instead of mounting the dir? Am I missing something? Let's not block on this, though.

Reply:
Good question, it's a pattern I copied from elsewhere, I added a rationale in https://github.com/osbuild/bootc-image-builder/pull/277 - does that make sense?